src/test/v2/common/steps_file.ts

```diff
@@ -113,5 +114,11 @@ export = function() {
       }
       return rsp;
     },
+    checkA11y(I: any, fileName: string) {
+      I.runA11yCheck({ reportFileName: fileName });
```

Comment:
You shouldn't be passing "I" in here - all references to I should work by just using "this."